x-pack/plugins/cases/server/client/configure/client.ts

```diff
@@ -368,6 +374,12 @@ export async function create(
     validateDuplicatedCustomFieldKeysInRequest({
       requestCustomFields: validatedConfigurationRequest.customFields,
     });
+    validateRequiredCustomFieldsInRequest({
+      requestCustomFields: validatedConfigurationRequest.customFields,
+    });
+    validateOptionalCustomFieldsInRequest({
+      requestCustomFields: validatedConfigurationRequest.customFields,
+    });
```

Comment:
It seems like these two can be merged into one function called for example `validateDefaultValue`. They code in the functions seems identical. Wdyt?